src/build.sh

```diff
@@ -97,6 +100,10 @@ function kernel_build()
     llvm='LLVM=1 '
   fi
 
+  if [[ -n "$clean" ]]; then
+    build_clean "$output_kbuild_flag"
```

Comment:
You probably want:
1. Check if `build_clean` worked as expected or not; in case of error show an error message;
2. Return after this function.